app/controllers/posts_controller.rb

```diff
@@ -6,4 +6,25 @@ def index
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @id = current_user.id
+    @post = Post.new
+  end
+
+  def create
+    post = Post.new(title: post_params[:title], text: post_params[:text], user: current_user)
```

Comment:
- [X] you can use `@post` already initialized in the new method

Reply:
Thank you for your positive insights